### ISSUE TITLE
Question about datasat?

### ISSUE THREAD

**Issue by Qmebius** [apple/ml-neuman#99]
I would like to ask if the dataset provides the 2D keypoints of SMPL.  I have downloade but not found

**Comment by jiangwei221** [comment]
We used mmpose to detect the 2D keypoints: https://github.com/apple/ml-neuman/blob/0149d258b2afe6ef65c91557bba9f874675871e4/preprocess/gen_run.py#L95-L104

**Comment by Qmebius** [comment]
Thanks，I would also like to ask, the bin files provided in the dataset contain depth information, which is very sparse in the human body area. Is there any way to obtain the depth information of the human body?

**Comment by jiangwei221** [comment]
Yes, we run BoostingMonocularDepth to estimate the human depth and fuse with colmap MVS depth.
1. BoostingMonocularDepth:
    https://github.com/apple/ml-neuman/blob/0149d258b2afe6ef65c91557bba9f874675871e4/preprocess/gen_run.py#L106-L121
2. Depth fusion:
    https://github.com/apple/ml-neuman/blob/0149d258b2afe6ef65c91557bba9f874675871e4/data_io/neuman_helper.py#L77-L86

**Comment by Qmebius** [comment]
And I also want to ask how you obtain the ground truth of the camera pose, and is the 'trans' processed by alignment.py the gt value of the human body's position?

**Comment by jiangwei221** [comment]
> And I also want to ask how you obtain the ground truth of the camera pose, and is the 'trans' processed by alignment.py the gt value of the human body's position?

We don't have a "ground truth camera pose" which usually requires manual annotations or extra hardwares. We use COLMAP to reconstruct the "reference camera poses".

**Comment by Qmebius** [comment]
So can this reference pose be used as the ground truth for the camera tracking task？



---Original---
From: "Wei ***@***.***&gt;
Date: Wed, Jul 10, 2024 01:00 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [apple/ml-neuman] Question about datasat? (Issue #99)




  
And I also want to ask how you obtain the ground truth of the camera pose, and is the 'trans' processed by alignment.py the gt value of the human body's position?
  
We don't have a "ground truth camera pose" which usually requires manual annotations or extra hardwares. We use COLMAP to reconstruct the "reference camera poses".

**Comment by MikeAiJF** [comment]
> > 另外想问一下你是怎么得到相机姿态的ground truth的，还有alignment.py处理后的trans是不是人体位置的gt值呢？
> 
> 我们没有“地面真实相机姿势”，这通常需要手动注释或额外的硬件。我们使用 COLMAP 来重建“参考相机姿势”。

你好，请问我应该怎么才能得到4d-humans文件下的东西
